docs/src/io.md

```diff
@@ -21,6 +21,15 @@ MPI.File.sync
 
 ## Data access
 
+### Views
```

Comment:
```suggestion
### Individual pointer
```
The MPI spec calls these routines "individual file pointers": https://www.mpi-forum.org/docs/mpi-3.1/mpi31-report/node319.htm#Node319